reading/README.md

```diff
@@ -9,11 +9,11 @@ There is a ton of links to extra reading and videos through out the material. Oc
 
 ### Blogs
 
-[Going Go](http://www.goinggo.net) - William Kennedy  
-[The acme of foolishness](http://dave.cheney.net) - Dave Cheney  
+[Ardan Labs Blog](https://www.ardanlabs.com/blog/) - William Kennedy  
```

Comment:
Hi @ardan-bkennedy 
I'm not sure about this change, could you please take a look should it be updated?
Thx!